HHS/Head-Start-TTADP
# [TTAHUB-1095] Limit email notification options

## Description of change

This change has 4 parts.

1. Add new user roles: ITM, IST, GMS
2. Limit what email notification e-mails can be seen/set per user roles (see JIRA).
3. Hide and show the TR checkbox based on user roles (see JIRA). 
4. Write a migration to remove email settings (UserSettingOverrides) if the role is invalid.

## How to test

1. Make sure the new roles are usable.
2. Check the appropriate e-mail settings show based on the JIRA spec.
3. Create a TR and verify the check box only shows with the roles on the JIRA spec.
4. Review and run the migration
Note: changing the delete to a select at the bottom of each query will let you see what's going to be deleted. 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1095


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
